src/core/lib/surface/call.h

```diff
@@ -265,6 +265,10 @@ void grpc_call_log_batch(const char* file, int line, const grpc_op* ops,
 
 void grpc_call_tracer_set(grpc_call* call, grpc_core::ClientCallTracer* tracer);
 
+// Both sets and manages call tracer life cycle using ClientCallTracerWrapper.
+void grpc_call_tracer_set_and_manage(grpc_call* call,
```

Comment:
Let's add a comment saying something to the effect of - Used by Python as a convenient way of using the arena to manage the lifetime of the call tracer.

Also add a comment explaining why we couldn't use `arena->ManagedNew<>` directly in Python